src/System.Reflection.Metadata/tests/Metadata/HandleTests.cs

```diff
@@ -541,7 +541,7 @@ public void NamespaceKinds()
             Assert.False(full.IsVirtual);
             Assert.Equal(123, full.GetHeapOffset());
 
-            var synthetic = NamespaceDefinitionHandle.FromSimpleNameOffset(123);
+            var synthetic = NamespaceDefinitionHandle.FromSyntheticId(123);
```

Comment:
@tmat do you think we should remove the Synthetic designation  bit and make these use the virtual bit. It feels more like what we use virtual for elsewhere now.